redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+// ACTIONS
+const CHECK_STATUS = 'bookstore/src/redux/category/CHECK_STATUS';
+
+// REDUCERS
+const categories = (state = [], action) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'UNDONE';
```

Comment:
- [ ] Kindly update the returned text in `check status` case to `Under Construction` as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md#:~:text=For%20the%20check%20status%20action%20return%20a%20string%20%22Under%20construction%22.).